### DISCUSSION TITLE
Questions about training

### DISCUSSION THREAD

**Discussion by KatieBelli** [voicepaw/so-vits-svc-fork#613] [Q&A]
I created 77 singing audio files and 3 speaking audio files (every file's length: <10 seconds).
This is my first dataset (and currently the only one I created).

My questions are:

01. When I train it shows that it seems to be training with only 38 files or what does the number 38 mean?
![SC02](https://github.com/voicepaw/so-vits-svc-fork/assets/109644887/c65c6fb4-93e5-4154-9e23-91425503370e)

02. I definitely want to add more audio files in the near future to the dataset which I currently train.
Do I need to add them to the existing dataset and start a completely new training?
Or can I just put them to the existing dataset and continue training (if yes, please tell me the steps to add them to the existing dataset)?

03. How many epochs are recommended for good results (please don't tell how many steps because this confuses me)?

**Comment by Mixeykusje** [comment]
The 38 is the number of steps to create 1 epoch. 
You have a total of 80 samples and your number of steps is 38, so your batchsize would be approximately between 2 and 3

**Comment by mynumu** [comment]
Hello. I am also trying to understand this setting in config.json

Please can someone ELI5 what you need to set in config.json based on sample sizes. First attempt. I have 60 samples x 10 seconds (all wavs) Data is around 471mb (but I dont see that data size makes any difference?)

How do you work out what to change in the config.json based on sample size?

Thanks

**Reply by Grownz** [reply]
What is your Question, exactly? How many Epochs you need, or how epochs, steps and batches are linked?

**Reply by mynumu** [reply]
Hi

all of the above please. Also tried this on colab but the epoch config reset to 10000 runs for the above dataset. Colab times out. TIA
